src/test/java/org/takes/rq/RqMultipartTest.java

```diff
@@ -494,22 +553,31 @@ public void producesPartsWithContentLength() throws IOException {
                 ),
                 Joiner.on("\r\n").join(
                     "--AaB01x",
-                    "Content-Disposition: form-data; name=\"t2\"",
+                    String.format(
+                        RqMultipartTest.CONTENT,
+                        part
+                    ),
```

Comment:
@raphaelln join lines here as well, please do it always when possible (it's not automatically enforced yet)